src/components/home/FindUs.tsx

```diff
@@ -1,5 +1,36 @@
-const FindUs = () => {
-  return <div></div>;
+"use client";
+import Image, { StaticImageData } from "next/image";
+
+interface FindUsProps {
+  title: string;
+  info: string;
+  image1: StaticImageData;
+  image2: StaticImageData;
+}
+
+const FindUs = ({ title, info, image1, image2 }: FindUsProps) => {
+  return (
+    <div className="bg-vsa-yellow-100 flex flex-col items-start justify-center p-20">
+      <div className="text-vsa-brown font-vsa-main p-4 pl-60 text-left text-5xl font-bold">
```

Comment:
```suggestion
      <div className="text-vsa-brown font-vsa-main p-4 text-left text-5xl font-bold">
```
use a fractional width here instead to avoid responsiveness issues